ocs_ci/helpers/cephx_keyrotation_helper.py

```
@@ -0,0 +1,4512 @@
+        """
+        Read the OSD CephX key from the running OSD pod keyring file.
+
+        Args:
+            osd_id (int): OSD id.
+
+        Returns:
+            str: Base64 CephX key for ``osd.<id>``.
+        """
+        osd_pods = get_osd_pods_having_ids([osd_id])
+        if not osd_pods:
+            raise UnexpectedBehaviour(f"No running OSD pod found for osd.{osd_id}")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)